### PR TITLE
Implement all the required changes for this project.

### PR DESCRIPTION
**Pull request by IsmailChangezi**
## Description 
- Add all the required features
- Use git work flow 
- Style the pages 
- Fix Linter Errors
- Install the required project dependencies

### CODE REVIEW

File: src/components/Profile.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          <li key={rocket.id} className="p-1 text-2xl border">
+            {rocket.rocket_name}
+          </li>
+        ))}
+      </ul>
+      <ul className="w-50">
+        <h1>My Missions</h1>
+        {reservedMissions.map((mission) => <li key={mission.id} className="p-1 text-2xl border">{mission.name}</li>)}
+      </ul>
+    </div>
```
Comment (VICTOR-CHUKWU) [inline src/components/Profile.js:24]: - [ ] Good job team, you know you can make your profile more interactive by adding a message when `reservedRockets` and `reservedMissions` is empty :+1:

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+function App() {
+  return (
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Rockets />} />
+        <Route path="/Missions" element={<Missions />} />
+        <Route path="/Profile" element={<Profile />} />
+      </Routes>
+    </>
```
Comment (VICTOR-CHUKWU) [inline src/App.js:16]: - [ ] nice understanding react Router

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @IsmailChangezi  and @Ofarouq310 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] good use of React Router

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Profile.js (hunk 1 above)
Comment (Ofarouq310) [reply src/components/Profile.js:24]: Thank you, Sir, for raising our attention to that detail.

File: src/App.js (hunk 2 above)
Comment (Ofarouq310) [reply src/App.js:16]: We appreciate your compliment a lot, Sir.

**Pull request merged**